doc/autoscale/experiment/README.md

```diff
@@ -158,3 +172,22 @@ serving together.
         ```
     1. Gernerate Experiment Report
         After all the passes are finished, the report will generated at './out' folder.
+
+
+## Conclusions
+
+### Resource utilization increased in both cases
+
+Utilization increased by XX% in case one, and XX% in case two
+
+### Average Pending time reduced in both cases
+
+XX% in case one and XX% in case two.
+
+### Average execution time reduced in both cases
+
+XX% in case one and XX% in case two.
+
+### Improved the service quality with Hybrid cloud
+
+As shown in test case two, Paddlepaddle yields resource to more important online services when QPS is get intensive.
```

Comment:
is get -> is getting

Maybe change "QPS" to "the load", since our test case 2 does not use QPS (uses pod count instead).

Reply:
got it, thanks!